CNV-27866: [Doc] expose the downwardMetrics through a virtio-serial channel (#74920)
openshift/openshift-docs: modules/virt-enabling-disabling-downward-metrics-feature-gate-cli.adoc

// Module included in the following assemblies:
//
// * virt/monitoring/virt-exposing-downward-metrics.adoc

:_mod-docs-content-type: PROCEDURE

> [id="virt-enabling-disabling-downward-metrics-feature-gate-cli__{context}"]

Reviewer: I think there's an extra underscore here that can be deleted:
```suggestion
[id="virt-enabling-disabling-downward-metrics-feature-gate-cli_{context}"]
```

Reviewer: This one item from my peer review still needs to be addressed.

= Enabling or disabling the downward metrics feature gate from the command line

> To expose downward metrics for a host virtual machine, you can enable the `downwardMetrics` feature gate by using the command line. Enabling the feature gate requires administrator privileges.

Reviewer: I would suggest making the last sentence a prereq:
```suggestion
To expose downward metrics for a host virtual machine, you can enable the `downwardMetrics` feature gate by using the command line. 

.Prerequisites

* You have access to the cluster with administrator privileges.
```
